doc/functions.md

```diff
@@ -121,9 +129,22 @@ def times2 x := x * 2
 #eval 100 |> add1 |> times2
 #eval times2 <| add1 <| 100
 ```
-The result of the previous `#eval` commands is 202.
-The forward pipeline `|>` operator takes a function and an argument and return a value.
-In contrast, the backward pipeline `<|` operator takes an argument and a function and returns a value.
+The result of all three previous `#eval` commands is 202.
+
+So `x |> f` means the same as the same as `f x`, and it chains such that `x |> f |> g` is
+interpreted as `g (f x)`.
+
+Similarly `f <| x` means the same as the same as `f x`, except that it parses x with lower precedence,
+which means that `f <| g <| x` is interpreted as `f (g x)` rather than `(f g) x`.
```

Comment:
Similarly `f <| x` means the same as the same as `f x`, ~except that it parses x with lower precedence,
which means that~, and `f <| g <| x` is interpreted as `f (g x)` ~rather than `(f g) x`.~

Reply:
No issue, just suggestions from reading it. 
Also, I think that ‘parses with lower precedence’ here is really intending to say ‘is right associative’ because the lower precedence is just an implementation detail of a precedence climbing/Pratt expression parser.

Thanks!

Reply:
The precedence is important for more than the associativity of `<|` itself, it also comes up when comparing `<|` to other operators like `+` - `f <| x + y` means `f (x + y)` and not `f x + y`.